.flake8

```diff
@@ -7,4 +7,5 @@ select = B,C,E,F,W,T4,B9,Q0,N8,VNE
 exclude =
     **migrations
     venv
-    tests
\ No newline at end of file
+    tests
+    settings.py
```

Comment:
why do you need to add settings.py here?

Reply:
because flake argues on AUTH_PASSWORD_VALIDATORS and SECRET_KEY